### PR TITLE
Added code to change Lights

### LINKED ISSUE #162: Configure the PiHat Led's with SUSI's Response
**Actual Behaviour**

Currently, no lights are configured with SUSI.AI

**Expected Behaviour**

To configure the PiHat's LED in Sync with SUSI's response

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #162 

Added Transitions Similar to Google Home which change color on every change of state

### CODE REVIEW

File: main/states/lights.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    def _off(self):
+        self.write([0] * 3 * self.LIGHTS_N)
+
+    def write(self, colors):
+        for i in range(self.LIGHTS_N):
+            self.dev.set_pixel(i, int(colors[3*i]), int(colors[3*i + 1]), int(colors[3*i + 2]))
+
+        self.dev.show()
+
+lights = Lights()
```
Comment (hongquan) [inline main/states/lights.py]: Again???

Comment (stealthanthrax) [reply main/states/lights.py]: @hongquan , updated it .

**Comment by hongquan** [comment]
@stealthanthrax TravisCI is crying about some PEP8 errors. Please use pycodestyle at your localhost to check new files you added in this PR.

**Comment by stealthanthrax** [comment]
Thanks @hongquan , I have updated it now

**Pull request merged**